frontend/src/components/ApolloAuthProvider.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  ApolloClient,
+  ApolloLink,
+  ApolloProvider,
+  createHttpLink,
+  fromPromise,
+  HttpLink,
+  InMemoryCache,
+} from '@apollo/client'
+import { useAuth0 } from '@auth0/auth0-react'
+
+import { FunctionComponent } from 'react'
+
+const clientUrl = process.env.REACT_APP_CLIENT_URL
+
+// https://www.apollographql.com/docs/link/links/http/
+const httpLink = new HttpLink({
+  uri: process.env.REACT_APP_SERVER_URL,
+  credentials: 'include',
+})
+
+const ApolloAuthProvider: FunctionComponent = ({ children }) => {
```

Comment:
Refactored the Apollo client instance into a wrapper component that's then used in `AppRoot`.